cluster-up/cluster/external/provider.sh

```diff
@@ -5,11 +5,17 @@ function _kubectl() {
 }
 
 function prepare_config() {
-    cat >hack/config-provider-external.sh <<EOF
-docker_tag=devel
-docker_prefix=${DOCKER_PREFIX}
-manifest_docker_prefix=${DOCKER_PREFIX}
-image_pull_policy=${IMAGE_PULL_POLICY:-Always}
+    BASE_PATH=${KUBEVIRTCI_CONFIG_PATH:-$PWD}
+
+    # required for running tests within openshift ci, otherwise tests don't find the configuration
+    ln -f -s ${KUBECONFIG:-$HOME/.kube/config} ${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
```

Comment:
Could you wrap the expressions here with quotes? Though unlikely, but if KUBECONFIG will have a space in its path this command will fail.

I'd suggest quoting always unless you explicitly want bash to split your expression

Reply:
Yeah, thought so too. I'd rather fix all `shellcheck` warnings but didn't want to go too far here ;-)

Reply:
Done.